doc/tutorials/application-integration.md

```diff
@@ -205,39 +200,39 @@ For more information about that, continue reading
 
 ## Specification ##
 
-Now, we have a fully working configuration system without any hard-coded
-information (such as configuration files). So we already gained something.
+Now, we have a fully working configuration system without any hardcoded
```

Comment:
Right, so we should still decide for a common style for our usage and not mix it up. a quick grep indicates that apart from some news that probably don't matter hardcode or its variations are only used in the application integration tutorial, so its hard to find a majority to follow here.

The tool is a linter, so all it does is complaining :)